schema/gendummydata.py

```diff
@@ -41,8 +53,23 @@
 RANDOM_TLDS = ("edu", "com", "org", "net", "tw", "ru", "pl", "de", "es")
 RANDOM_URL = ("http://www.", "ftp://ftp.", "http://", "ftp://")
 RANDOM_LOCS = ("pub", "release", "files", "downloads", "src")
-FORTUNE_FILE = os.environ.get("FORTUNE_FILE", "/usr/share/fortune/cookie")
 
+if (os.name == 'posix'):
+    OS = getosname()
+    if OS == 'arch':
+        FORTUNE_FILE = "/usr/share/fortune/cookie"
+    elif OS == 'debian':
+        FORTUNE_FILE = "/usr/share/games/fortunes/fortune"
+    elif OS == 'ubuntu':
+        # Needs to be verified.
+        FORTUNE_FILE = "/usr/share/games/fortunes/fortune"
+    elif OS == 'fedora':
+        FORTUNE_FILE = "/usr/share/games/fortunes/fortune"
+    else:
+        print(f"Please check and Install Fortune for your os: {sys.platform} and set path for FORTUNE_FILE env var.")
+        FORTUNE_FILE = os.environ.get("FORTUNE_FILE")
+
+FORTUNE_FILE = os.environ.get("FORTUNE_FILE", FORTUNE_FILE)
```

Comment:
If the file is unable to be found, could we prompt the user to install the needed package? I was thinking just setting something like `PKGNAME` in each if/elif block and then prompting if needed from there.

Reply:
Yes that bit of code is already in there. https://github.com/makedeb/mprweb/blob/b050c408641c05521c47465e4f56cfb49f337d23/schema/gendummydata.py#L63